cmake/configure_s4.intel.cmake

```diff
@@ -0,0 +1,4 @@
+set(INLINE_POST     ON CACHE BOOL "Enable inline post" FORCE)
+set(AVX2            OFF CACHE BOOL "Enable AVX2 instruction set" FORCE)
+set(IVYBRIDGE       ON CACHE BOOL "Enable AVX2 instruction set" FORCE)
```

Comment:
I think that `IVYBRIDGE` (or preferably `AVX`) should be added to the top-level `CMakeLists.txt` as well.

Reply:
I've made the change here to `AVX` as well as the top-level `CMakeLists.txt`.